sqlglot/parser.py

```diff
@@ -4466,6 +4466,15 @@ def _parse_ordered(
             exp.Ordered, this=this, desc=desc, nulls_first=nulls_first, with_fill=with_fill
         )
 
+    def _parse_limit_options(self) -> exp.LimitOptions:
+        percent = self._match(TokenType.PERCENT)
+        rows = self._match_set((TokenType.ROW, TokenType.ROWS))
+        only = self._match_text_seq("ONLY")
+        with_ties = self._match_text_seq("WITH", "TIES")
+        if only and with_ties:
+            self.raise_error("Cannot specify both ONLY and WITH TIES in FETCH clause")
```

Comment:
Probably not necessary to do error-checking here, i.e. we can remove L4472, L4474, L4475.